uber/site_sections/preregistration.py

```diff
@@ -67,6 +67,33 @@ def stats(self):
     def check_prereg(self):
         return json.dumps({'force_refresh': state.AFTER_PREREG_TAKEDOWN or state.BADGES_SOLD >= MAX_BADGE_SALES})
 
+    def check_if_preregistered(self, session, message="", **params):
+        if 'email' in params:
+            attendee = session.query(Attendee).filter(func.lower(Attendee.email) == func.lower(params['email'])).one()
+            message = "Thank you! You will receive a confirmation email if you are registered for "+EVENT_NAME_AND_YEAR+"."
+            subject = EVENT_NAME_AND_YEAR+' Registration Confirmation'
+            email_text = '''<html>
```

Comment:
When I tried to use a template with send_email, it didn't render. Unless there's a reasonably easy way to fix that, we'll unfortunately have to keep the email text inside this function for now.